### ISSUE TITLE
scrolling issue with nvim < 0.4dev

### ISSUE THREAD

**Issue by equalsraf** [equalsraf/neovim-qt#564]
When using ext_linegrid with a split, followed by a call to the wild menu leaves a duplicate status bar. Seems like an off by one scrolling issue.

![extlinegrid](https://user-images.githubusercontent.com/715238/62802005-dc1e7380-bade-11e9-9f8d-e8d1c3c0d623.png)

**Comment by jgehrig** [comment]
Thanks! I will take a look at this...

**Comment by jgehrig** [comment]
@equalsraf I am unable to reproduce this. I don't see the duplicate line when wildmenu is triggered from a split.

Could you have more specific steps? Do your specific vimrc settings have any impact?

**Comment by equalsraf** [comment]
Here we go, minimal vimrc

```
" test.vim
color blue                                 " i dont think this is important but #569 
set wildmenu
set wildmode=list:longest,full		" set wildmode to list full
set laststatus=2			" Always show statusline
```

To reproduce

1. open nvim-qt -- -u test.vim
2. horizontal split with `sp`
3. `:e <TAB><TAB><TAB>` i.e. go around the wildmenu completion, in my case this causes a scroll up
4. now move to the lower window with `C-w j`

**Comment by jgehrig** [comment]
Thanks @equalsraf!

I think you might have found a bug in Neovim... I can reproduce this with ext_linegrid off, and `nvim` TUI. 

Here is the issue outside of Neovim-Qt:
![image](https://user-images.githubusercontent.com/11207308/62990074-86b4d000-be18-11e9-834c-a074767501ac.png)

The second <TAB> stoke for `wildmenu=full` causes a shift, and the laststatus line is painted again (in the correct location) when you perform the `C-w j`.

This bug does not reproduce in vim (vimrc needed modification for vim/neovim compatibility):
```
set nocompatible " Needed for vim to use wildmenu?
set wildmenu
set wildmode=list:longest,full
set laststatus=2
```

**Comment by equalsraf** [comment]
Retested this with the master branch from neovim and i cannot reproduce.

**Comment by jgehrig** [comment]
Assuming this is closed.

I can confirm that this does not reproduce in NeovimQt + Neovim v0.4.0-1726-ge56f62e9a.
 
It looks like this issue has been fixed in master some time since release v0.3.7.

**Comment by equalsraf** [comment]
Ack, closing.
